tests/chainer_tests/functions_tests/array_tests/test_spatial_transformer_sampler.py

```diff
@@ -0,0 +1,125 @@
+import unittest
+
+import numpy
+
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+def _identiy_grid(in_shape):
+    mesh = numpy.meshgrid(
+        numpy.linspace(-1., 1., num=in_shape[2]),
+        numpy.linspace(-1., 1., num=in_shape[3]))
+    grid = numpy.stack(mesh).astype(numpy.float32)  # (2, H, W)
```

Comment:
Test results indicate that numpy.stack is not available in numpy v1.9. Please use another method.